test/graphql/queries/profile_query_test.rb

```diff
@@ -274,7 +274,7 @@ class ProfileQueryTest < ActiveSupport::TestCase
     (host3 = hosts(:two).dup).update!(name: 'host3')
     policies(:one).update(compliance_threshold: 95,
                           account: accounts(:test),
-                          hosts: [hosts(:one), hosts(:two), host3])
+                          hosts: [hosts(:one), host3])
```

Comment:
Creating the scenario where we have test results for a host but it's not assigned to the policy.